.github/ISSUE_TEMPLATE/release-runtime.md

```diff
@@ -6,21 +6,24 @@ title: Release Checklist for Runtime {{ env.VERSION }}
 
 # Release Checklist - Runtimes
 
+**All** following checks must be completed before publishing a new release.
+The release process is owned and led by @paritytech/release-engineering team.
+The checks marked with :crab: are meant to be checked by [a runtime engineer](https://github.com/paritytech/cumulus/issues/1761).
+
 ## Runtimes Release
 
 ### Codebase
 These checks should be performed on the codebase.
 
 - [ ] the [`spec_version`](https://github.com/paritytech/cumulus/blob/master/docs/release.md#spec-version) has been incremented since the
     last release for any native runtimes from any existing use on public (non-private/test) networks
-- [ ] previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are
-    removed for any public (non-private/test) networks
-- [ ] No migrations added in the last release that would need to be removed
+- [ ] :crab: previously [completed migrations](https://github.com/paritytech/cumulus/blob/master/docs/release.md#old-migrations-removed) are removed for any public (non-private/test) networks
 - [ ] pallet and [extrinsic ordering](https://github.com/paritytech/cumulus/blob/master/docs/release.md#extrinsic-ordering--storage) as well as `SignedExtension`s have stayed
     the same. Bump `transaction_version` otherwise
 - [ ] the [benchmarks](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus) ran
 - [ ] the weights have been updated for any modified runtime logic
-- [ ] the various pieces of XCM config are sane
+- [ ] :crab: the new weights are sane, there are no significant (>20%) drops or rises with no reason
+- [ ] :crab: the various pieces of XCM config are sane
 
 ### On the release branch
 
```

Comment:
```suggestion
### On the release branch

- [ ] :crab: Doublecheck that the release branch is not pointing to master for cumulus/polkadot/substrate.
```

Reply:
At some point we can take out this check but for now this is something to be checked as it's not yet automated.